src/toDoList.js

```diff
@@ -107,10 +119,23 @@ export default class ToDoList {
       toDo.append(removeBtn);
       return container.append(toDo);
     });
+    if (checked >= 1) {
+      const clearAll = document.getElementById('clear-all');
+      if (clearAll.classList.contains('disabled')) {
+        clearAll.classList.remove('disabled');
+      }
+    } else {
+      const clearAll = document.getElementById('clear-all');
+      if (!clearAll.classList.contains('disabled')) {
+        clearAll.classList.add('disabled');
+      }
+    }
   }
 
   clearAllCompleted(container) {
     container.innerHTML = '';
-    this.toDoList = [];
+    const newArray = this.toDoList.filter((item) => !item.completed);
+    this.toDoList = newArray;
+    this.update(container);
```

Comment:
- [ ] As requested in the project requirements, kindly make sure that you use Local Storage to store and retrieve your array,

Reply:
This is also loading and Setting the array to localStorage,